api/api/error_handler.py

```diff
@@ -55,6 +55,46 @@ def _cleanup_detail_field(detail: str) -> str:
     return ' '.join(str(detail).replace("\n\n", ". ").replace("\n", "").split())
 
 
+async def handle_expect_header(request: ConnexionRequest, exc: ContentSizeExceeded = None) -> ConnexionResponse:
+    """Handler for the 'Expect' HTTP header.
+    
+    Parameters
+    ----------
+    request : ConnexionRequest
+        Incoming request.
+    exc : ContentSizeExceeded
+        Raised exception if content size exceeds the limit.
+
+    Returns
+    -------
+    Response
+        HTTP Response returned to the client.
+    """
+    problem = {
+       "title": "Expectation failed",
+       "detail": "Unknown Expect",
+       "error": 417
+    }
```

Comment:
This could be defined inside the `if expect_value != continue_msg or (exc and expect_value == continue_msg):` condition because it's not used elsewhere.

Also, can we use `WazuhError` like the rest of the code instead of harcoding the values?

Reply:
https://github.com/wazuh/wazuh/issues/21935#issuecomment-1991758569